### PR TITLE
Undo bump to 0.1.4

### PR DESCRIPTION
**Pull request by dcharkes**
Package stable versions should only depend on sdk stable versions.

### CODE REVIEW

File: example/main.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import 'dart:ffi';
```
Comment (mkustermann) [inline example/main.dart]: License header.

Comment (dcharkes) [reply example/main.dart]: Adding a license header will make that header show up on the pub website. For example in package:http it is omitted: https://pub.dev/packages/http#-example-tab-.

**Comment by mit-mit** [comment]
Rather than undoing this, I think we need update this to depend on 2.6 stable, and then publish 0.1.4 next week

**Comment by mkustermann** [comment]
As I argued before, we could take the `10 points` scoring hit on pub.dev and publish *right now* with a `sdk: ">= 2.6.0-dev.8.2". Once pub.dev uses the new stable the `10 points` will come back automatically.

Advantage: We can have all packages ready and usable *before* D26 is out the door - instead of trying to do all of this *after* D26 is out the door.

**Comment by mit-mit** [comment]
Martin and & discussed: In this case 2.6 is close, we can go ahead and depend on that. @dcharkes can you please update this repo and get the PR landed; then I'll run `pub publish` when 2.6 goes out.

**Comment by dcharkes** [comment]
Landing this, and opening a new PR that bumps `package:ffi` to `0.1.3` (stable) and depends on Dart `2.6.0` stable.

**Pull request merged**